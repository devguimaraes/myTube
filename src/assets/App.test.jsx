/* eslint-disable no-undef */
import { render } from '@testing-library/react';
import App from '../App';

describe('Snapshot', () => {
	test('Snpashot app', () => {
		const { container } = render(<App />);

		expect(container).toMatchSnapshot();
	});
});
