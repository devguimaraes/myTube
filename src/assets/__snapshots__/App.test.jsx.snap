// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Snpashot app 1`] = `
<div>
  <div
    class="App"
  >
    <h1>
      Ol√°, mundo!
    </h1>
  </div>
</div>
`;
